execution/function/absent.go

```diff
@@ -57,13 +57,19 @@ func (o *absentOperator) loadSeries() {
 
 		// https://github.com/prometheus/prometheus/blob/df1b4da348a7c2f8c0b294ffa1f05db5f6641278/promql/functions.go#L1857
 		var lm []*labels.Matcher
-		switch n := o.funcExpr.Args[0].(type) {
-		case *logicalplan.VectorSelector:
-			lm = append(n.LabelMatchers, n.Filters...)
-		case *logicalplan.MatrixSelector:
-			v := n.VectorSelector
-			lm = append(v.LabelMatchers, v.Filters...)
-		default:
+		logicalplan.TraverseBottomUp(nil, &o.funcExpr.Args[0], func(parent, node *logicalplan.Node) bool {
```

Comment:
Shouldn't this be in the logical plan?

Reply:
The sector labels of the vector sector of the absent function? I think we could put it into the functioncall node and populate it in the plan yeah